nginx-asg.tf

```diff
@@ -0,0 +1,194 @@
+data "aws_ami" "bgdc_nginx_latest" {
+  most_recent = true
+
+  filter {
+    name   = "name"
+    values = ["bgdc-nginx-ami-main-*"]
+  }
+
+  filter {
+    name   = "virtualization-type"
+    values = ["hvm"]
+  }
+
+  owners = [local.bgdc_account.test]
+}
+
+resource "aws_launch_configuration" "nginx_conf" {
+  name            = "dwx_bgdc_nginx"
+  image_id        = data.aws_ami.bgdc_nginx_latest.id
+  instance_type   = "t2.medium"
+  security_groups = [aws_security_group.nginx-bgdc-dwx.id]
+  iam_instance_profile = aws_iam_instance_profile.dwx_bgdc_nginx_instance_profile.arn
+ 
+  
+  user_data = templatefile("bootstrapBgdcDwxNginx.template", { BgdcDwxListener = 400, BgdcDwxNginxDns = "localhost", BgdcDwxNginxPort = 400 })
+
+  root_block_device {
+    volume_type           = "gp3"
+  }
+
+  lifecycle {
+    create_before_destroy = false
+  }
+
+}
+
+resource "aws_autoscaling_group" "nginx_asg" {
+  name                      = "dwx_bgdc_nginx"
```

Comment:
This name has underscores where as other resources have names with dashes in. Can we keep this consistent so its easier for when people refer to remote states